velox/connectors/hive/storage_adapters/hdfs/HdfsWriteFile.cpp

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "velox/connectors/hive/storage_adapters/hdfs/HdfsWriteFile.h"
+#include <hdfs/OutputStream.h>
+#include <hdfs/hdfs.h>
+
+namespace facebook::velox {
+
+HdfsWriteFile::HdfsWriteFile(hdfsFS hdfs, std::string_view path) {
```

Comment:
Replace this with

```
HdfsWriteFile(hdfsFS hdfs, std::string_view path) : HdfsWriteFile(hdfs, path, 0, 0, 0) {}
```

in the header file to avoid writing large comment that can get out of sync easily.